.github/workflows/publish-rc.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Publish Release Candidate'
+
+on:
+  push:
+    branches:
+      - main
+    # https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths
+    paths-ignore:
+      - '**/package.json'
```

Comment:
> When all the path names match patterns in `paths-ignore`, the workflow will not run

Trying to prevent the RC commit to `main` from re-triggering this action.

Reply:
How is this working exactly? does it prevent a loop from occurring after every push?

Reply:
Yeah, because every push is going to add the updated `package.json`s that the publish creates